react/src/components/AgentStats.tsx

```diff
@@ -0,0 +1,222 @@
+import { useResourceSlotsDetails } from '../hooks/backendai';
+import BAIFetchKeyButton from './BAIFetchKeyButton';
+import { useControllableValue } from 'ahooks';
+import { Segmented, Skeleton, theme, Typography } from 'antd';
+import {
+  BAIFlex,
+  BAIBoardItemTitle,
+  ResourceStatistics,
+  convertToNumber,
+  processMemoryValue,
+} from 'backend.ai-ui';
+import _ from 'lodash';
+import { useMemo, useTransition, ReactNode } from 'react';
+import { useTranslation } from 'react-i18next';
+import { graphql, useRefetchableFragment } from 'react-relay';
+import { AgentStatsFragment$key } from 'src/__generated__/AgentStatsFragment.graphql';
+
+interface AgentStatsProps {
+  queryRef: AgentStatsFragment$key;
+  isRefetching?: boolean;
+  displayType?: 'used' | 'free';
+  onDisplayTypeChange?: (type: 'used' | 'free') => void;
+  extra?: ReactNode;
+}
+
+const AgentStats: React.FC<AgentStatsProps> = ({
+  queryRef,
+  isRefetching,
```

Comment:
Although the loading indicator is displayed via `isRefetching`, the data used in the `Agentstats` component is not being refetched.